Makefile

```diff
@@ -2,23 +2,28 @@
 
 include Makefile.include
 
-env-up: env-compose-up env-devcontainer     ## Start devcontainer.
+env-up: env-update-image 			## Start devcontainer.
+	docker-compose up -d
 
-env-compose-up:
-	docker-compose pull
+env-up-rebuild: env-update-image	## Start devcontainer with rebuild.
+	docker-compose up --build -d
+
+env-update-image:					## Pull latest dev image
+	docker pull "perconalab/pmm-server:dev-latest"
+
+env-compose-up: env-update-image
 	docker-compose up --detach --renew-anon-volumes --remove-orphans
 
 env-devcontainer:
 	docker exec -it --workdir=/root/go/src/github.com/percona/pmm pmm-managed-server .devcontainer/setup.py
 
-env-down:                                   ## Stop devcontainer.
+env-down:							## Stop devcontainer.
```

Comment:
minor: all comments had the same indents

Reply:
they are indented 
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/91831719/179177532-1b080485-6199-4b34-8afc-983b5e9ac437.png">

Reply:
probably there is combination of tabs and spaces and that's why it's showing differently on your screen and in GitHub